src/app.js

```diff
@@ -37,5 +37,30 @@ app.get('/online-users', async (req, res) => {
   }
 })
 
+app.post('/create-game', async (req, res) => {
+  try {
+    const { users } = req.body
```

Comment:
I believe the property was supposed to be named `players`. It is important to stick to the specification, because in real projects, other people go by these requirements and implement accordingly. If at any point in the link something is not implemented to the specification, the whole communication chain breaks.
<img width="369" alt="Screen Shot 2022-09-23 at 11 20 05" src="https://user-images.githubusercontent.com/104974346/191919647-c5b58433-6cd3-4ee2-8b18-78b8111604d3.png">